src/SSL/Entities/CheckedCertificate.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace UKFast\SDK\SSL\Entities;
+
+class CheckedCertificate
+{
+    /**
```

Comment:
As we're surfacing the chain as well, it would be nice if we could use a report certificate entity which this entity has an array of

Reply:
parsed the chain and used a new entity for them